test/jenkins_tests/multi_node_tests/remove_driver_test.py

```diff
@@ -208,14 +208,9 @@ def try_to_create_actor(actor_class, driver_index, actor_index,
 
     # Only one of the cleanup drivers should create more actors.
     if driver_index == 2:
-        # Create some actors that require two GPUs.
```

Comment:
There will be random failure because when 1-GPU actors are assigned to 2-GPU or 4-GPU node, there will be a 2-GPU actor cannot get allocated and cause timeout.

Reply:
Example can be found here: https://amplab.cs.berkeley.edu/jenkins/job/Ray-PRB/7055/console.

Reply:
I don't really see how that could happen. Won't driver 0 always be able to finish? And then once driver 0 finishes, won't driver 1 always be able to place the 2-GPU actor?

Reply:
You can see this [log](https://amplab.cs.berkeley.edu/jenkins/job/Ray-PRB/7055/console). A 2-GPU actor of Driver 2 never gets assigned. 
```bash
Kill one process.
STDOUT:

STDERR:
The actor with ID eea4ae6175e7bbeb4a46f1a6ad20cf91d3a877a1 is taking a while to be created. It is possible that the cluster does not have enough resources to place this actor (this may be normal while an autoscaling is scaling up). Consider reducing the number of actors created, or increasing the number of slots available by using the --num-cpus, --num-gpus, and --resources flags. The actor creation task is requesting 1 CPU 2 GPU 
Suppressing duplicate error message.
Suppressing duplicate error message.
Suppressing duplicate error message.
...
```